docs/tutorials/05_torch_connector.ipynb

```diff
@@ -311,22 +310,15 @@
    ],
    "source": [
     "# Evaluate model and compute accuracy\n",
-    "y_predict = []\n",
-    "for x, y_target in zip(X, y):\n",
-    "    output = model1(Tensor(x))\n",
-    "    y_predict += [np.sign(output.detach().numpy())[0]]\n",
-    "\n",
+    "y_predict = np.sign(model1(X_).detach().numpy()).squeeze()\n",
+    "    \n",
```

Comment:
In general, the same comment, the line `np.sign(model1(X_).detach().numpy()).squeeze()` has a lot operations and may take some time to understand. I'm not saying that the original code is way better, but since we change it is nice to have make it really nice. Would be valuable to add a comment like "we evaluate model with a batch of data... " or something like that and explain why `squeeze` is required.